CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 1.2.0
+## 1.2.1
+
+- Label support for query varaible in dashboard
```

Comment:
```suggestion
- Feature: Add label support for query variable in dashboard
```

Reply:
We might also need to put a note here that existing dashboards might need to be updated incase that their variable query returned multiple columns before. 
The change should not brake anything since previously also the first column was used as the variable value, but cosmetic wise, with this update, different variable labels might be displayed in existing dashboards.